### PR TITLE
Balancetweak

### PR DESCRIPTION
**Pull request by buckly90**
Made some small tweaks to hopefully make the game more enjoyable.

- Limited all patch lighting to a maximum of 100% No more 150% or above.
- Buffed photosynthesis a little to make up for the lower limits of light.
- Rearranged membrane stats in an attempt to make each type more unique and viable.
- Buffed respiration parts. You can now support a flagellum with as little as two metabolosomes. (Your still gonna need alot if you want to be complex.)

These changes will be revisited once the spawn system is finalized and I am able to freely tweak compound rates. For now, the changes are aimed to hopefully make the game more enjoyable in it's current state.

### CODE REVIEW

**Comment by buckly90** [comment]
I'm not currently sure where the code for the membrane tooltips resides, and I need to change that so the editor displays the appropriate stats for membranes.

File: simulation_parameters/microbe_stage/biomes.json [hunk 1]
```diff
@@ -25,12 +25,12 @@
                     "name": "Floating Hazard",
                     "meshes": [
                         {
-                            "scenePath": "res://src/microbe_stage/ToxinCloud.tscn"
```
Comment (hhyyrylainen) [inline simulation_parameters/microbe_stage/biomes.json:28]: This file looks like all the changes are backwards, this reverts the change to moving to using particles for the toxins.

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]
I think the tooltips might be in ToolTipManager.tscn, though @Kasterisk probably knows better

**Comment by Kasterisk** [comment]
Yeah all the custom tooltips are currently in ToolTipManager.tscn. The modifier values are hardcoded on the tooltip gui so you need to tweak the the gui labels text to change them.

**Comment by buckly90** [comment]
Tooltips have been changed, I went ahead and converted the remaining decimal values to percentage as well, I also fixed the issue with my commit reverting the changes to toxins.

**Review by hhyyrylainen — APPROVED** [review]
Looks like there are no longer any problems with unintended changes being included.
@Kasterisk could you check the GUI changes?
Other than that we might want to hear balance feedback from other devs.

**Review by Kasterisk — APPROVED** [review]
Looks good

**Comment by hhyyrylainen** [comment]
I'll merge this to prepare for tomorrow's release candidate.

**Pull request merged**